code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -494,6 +494,12 @@
 		if((nearby_carbons.name == oldtarget_name) && (world.time < last_found + 100))
 			continue
 
+		//BUBBERSTATION CHANGE START: BEEPSKY IS A DINOSAUR NOW. CAN'T SEE YOU IF YOU DON'T MOVE.
+		if(nearby_carbons.next_move + (4 SECONDS) <= world.time && nearby_carbons.next_click + (4 SECONDS) <= world.time)
```

Comment:
What is next_click checked for? Does this reset the timer if you say, pull something from inventory etc.

Reply:
Generally if you interact with anything at all, next_click is used.